.changeset/chatty-icons-accept.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
I was not sure if I had to update the upgrade guide as the user doesn't have to take any steps, but just know that caching has been enabled by default.

We are updating documentation and also release notes. Please let me what you think.

Reply:
I think we should document the new default behavior it as it is one of the breaking changes.

Reply:
Documented in upgrade guide.